proto/meta.proto

```diff
@@ -593,6 +593,27 @@ service SystemParamsService {
   rpc SetSystemParam(SetSystemParamRequest) returns (SetSystemParamResponse);
 }
 
+message GetSessionParamsRequest {}
```

Comment:
Can we first unify the terms? Or is there any difference between "session config" and "session param"?

Reply:
I tried to use session config in frontend. But I guess it's ok to just use session params everywhere

Reply:
But let me rename it in the next pr.

Reply:
You probably won't want to review 200 lines of rename in this pr...